### PR TITLE
Updated for 1.20.3 & 1.20.4

### PR DESCRIPTION
**Pull request by StrangeOne101**
- Added the 1.20.3 and 1.20.4 versions
- Added some stuff to the gitignore for when running locally in your IDE

### CODE REVIEW

**Review by ShaneBeee — CHANGES_REQUESTED** [review]
Thanks for doing this, I was gonna do it today, but got busy.
Now I don't have to ;) 

Just a few tiny requests

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.shanebeestudios</groupId>
     <artifactId>McDeob</artifactId>
-    <version>2.3.1</version>
+    <version>2.3.2</version>
```
[the remark this answers is not in the document]
Comment (StrangeOne101) [reply pom.xml:9]: Sure

[the remark this answers is not in the document]
Comment (ShaneBeee) [reply pom.xml:9]: it came out this morning. They got it out QUICK

**Comment by StrangeOne101** [comment]
Do note that the mappings for 1.20.3 and 1.20.4 are the same, but this isn't a mistake. Seems that since 1.20.4 only fixed a single bug, they didn't need to produce a new mapping file.

I tested it just in case to make sure it still works, and it does. It still maps fine

**Comment by ShaneBeee** [comment]
> Do note that the mappings for 1.20.3 and 1.20.4 are the same, but this isn't a mistake. Seems that since 1.20.4 only fixed a single bug, they didn't need to produce a new mapping file.
> 
> I tested it just in case to make sure it still works, and it does. It still maps fine

duly noted, thank you

**Review by ShaneBeee — APPROVED** [review]

**Comment by ShaneBeee** [comment]
merged, and thank you :)

**Pull request merged**